### PR TITLE
Fix bad HTML

### PR DESCRIPTION
**Pull request by dshevtsov**
HTML checking with enabled HTMCheck reported about HTML issues.
This pull request fixes the reported issues.

### CODE REVIEW

File: api/rest/Resources/Orders/sales_orders.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h5><a name="RESTAPI-Resource-SalesOrders-HTTPMethod-GET-orders"></a>HTTP Method
 
 <div class="panel" style="border-width: 1px;"><div class="panelContent">
 <div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
-<div id="root">
+<div>
```
Comment (jeff-matthews) [inline api/rest/Resources/Orders/sales_orders.html]: What was the error here?

File: api/rest/testing_rest_resources.html [hunk 2]
```diff
@@ -33,43 +37,55 @@
...
 
-<b>NOTE</b>: Click the header with authorization data and click <b>Auto refresh</b> in the opened pop-up in order to generate new values for oauth_nonce, oauth_timestamp, and oauth_signature at each request.<br/>
-<img src="{{ site.baseurl }}/guides/m1x/images/rest_test3.png" style="border: 1px solid black"/></span></li>
-	<li>From the <b>Headers</b> drop-down, select <b>Custom Header</b>.<br/>
-<img src="{{ site.baseurl }}/guides/m1x/images/rest_test4.png" style="border: 1px solid black"/></span></li>
-	<li>In the <b>Request Header</b> window, enter "Content-Type" in the <b>Name</b> field and "text/xml" in the <b>Value</b> field (if you want to use the XML data format). To use the JSON request data format, enter application/json instead of the text/xml value.</li>
-	<li>Click <b>Okay</b>.<br/>
-<img src="{{ site.baseurl }}/guides/m1x/images/rest_test5.png" style="border: 1px solid black"/></span></li>
+	<b>NOTE</b>: Click the header with authorization data and click <b>Auto refresh</b> in the opened pop-up in order to
+	generate new values for oauth_nonce, oauth_timestamp, and oauth_signature at each request.<br />
```
Comment (jeff-matthews) [inline api/rest/testing_rest_resources.html]: This looks like bad formatting. May not affect how the HTML renders, but it's ugly.

File: api/rest/Resources/Orders/sales_orders.html (hunk 1 above)
Comment (dshevtsov) [reply api/rest/Resources/Orders/sales_orders.html]: Topics have multiple occasions of the `id="root"`.
@belbiy confirmed we do not use the `root` id at all, so I removed it from all topics as a redundant id.

File: api/rest/testing_rest_resources.html (hunk 2 above)
Comment (dshevtsov) [reply api/rest/testing_rest_resources.html]: This is not the only case, unfortunately. The purpose of this PR is to fix only cases with invalid HTML to be able to enable the HTML checking on the entire website.

**Pull request merged**